### ISSUE TITLE
Publish 0.4.0.2

### ISSUE THREAD

**Issue by mossprescott** [mr/ftp-client#12]
Hi, we need the multi-line fix and would like to get the package from hackage if possible. It seems like there’s not much activity at the moment, maybe you could publish the latest so the fixes are available?

Thanks for providing a useful tool!

**Comment by mr** [comment]
I just published the package as version 0.5.0.0. Let me know if there are any issues. Something that should be noted is that now exceptions are thrown on certain responses.

```
data FTPException
    = FailureRetryException FTPResponse --^ 4xx
    | FailureException FTPResponse --^ 5xx
    | UnsuccessfulException FTPResponse --^ not 2xx (aka unexpected 3xx)
    | BogusResponseFormatException FTPResponse --^ valid response, but data doesn't match what was requested (for instance, bad mlsx parse)
    | BadProtocolResponseException ByteString --^ nonsense response, probably didn't start with a code
    deriving (Show, Typeable)
```

I'll try to get better documentation at some point and maybe some tests, I didn't know if anyone was actually using it.

**Comment by mossprescott** [comment]
Great, thanks a lot! And congratulations on having a user 😉
